pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -52,6 +52,11 @@ const (
 	ovsExternalIDNodeName = "node-name"
 
 	nodeRouteInfoPodCIDRIndexName = "podCIDR"
+
+	PodIPv4CIDRMaskSize = 24
+	IPv4BitLen          = 32
+	PodIPv6CIDRMaskSize = 64
+	IPv6BitLen          = 128
```

Comment:
These could be gathered runtime from local podCIDRs. Hardcoded values may work fine for now.. just want to confirm with @antoninbas

Reply:
It's fine to assume fixed size CIDRs across the cluster for now (because this is how Node IPAM works), but it is **not** fine to assume that the size is 24 (for IPv4) or 64 (for IPv6). The cluster admin can easily change this by providing a different value with `--node-cidr-mask-size` for [kube-controller-manager](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/).

Reply:
Thanks, Antonin.

Reply:
Thanks, I add the logic of gathering PodIPv4/v6CIDRMaskSize from local podCIDRs, if it doesn't exist, using 24 (for IPv4) or 64 (for IPv6) as default value instead.